plasmapy/formulary/tests/test_drifts.py

```diff
@@ -1,11 +1,68 @@
 import astropy.units as u
+import numpy as np
 import pytest
 
 from astropy.tests.helper import assert_quantity_allclose
 
 from plasmapy.formulary import drifts
 
 
+class Test_diamagnetic_drift:
+    q = 1 * u.C
+    n = 1 / u.m ** 3
+
+    def test_isothermal_plasma(self):
+        r"""
+            Cylindrical & isothermal plasma in Fig 3.4 of Chen.
```

Comment:
I was going to suggest putting this note in here. Oh well, love to be out of work in these cases :smile: